recipe/bld.bat

```diff
@@ -9,9 +9,6 @@ nmake /f makefile.vc
 nmake INSTDIR=%LIBRARY_PREFIX% /f makefile.vc install-all
 if errorlevel 1 exit 1
 
-move %LIBRARY_PREFIX%\bin\*.* %PREFIX%
```

Comment:
I am not sure why we had this line here but it is copying more than it should. Also, ` %LIBRARY_PREFIX%\bin` is in the path, so we don't really need it.

ping @gillins